upup/pkg/fi/cloudup/awsup/aws_utils.go

```diff
@@ -38,7 +38,11 @@ func ValidateRegion(region string) error {
 		glog.V(2).Infof("Querying EC2 for all valid regions")
 
 		request := &ec2.DescribeRegionsInput{}
-		config := aws.NewConfig().WithRegion("us-east-1")
+		awsRegion := os.Getenv("KOPS_AWS_REGION")
```

Comment:
Is this a new ENV VAR that we will need to be supporting?

Is this documented anywhere?